charts/app-config/values-integration.yaml

```diff
@@ -280,6 +280,13 @@ govukApplications:
       enabled: false
     dbMigrationEnabled: true
     workerEnabled: true
+    workers:
+      - command: ["bundle", "exec", "sidekiq", "-C", "config/sidekiq.yml"]
```

Comment:
optional: with the new base images, the `bundle exec` should now be optional (but I haven't actually tried it with sidekiq yet)

Reply:
cool, i'll take it out and give it a try with this then